espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/CreateSession.kt

```diff
@@ -38,11 +39,13 @@ class CreateSession : RequestHandler<SessionParams, Session> {
             val shouldLaunchApp = (parsedCaps["autoLaunch"] ?: true) as Boolean
             if (shouldLaunchApp) {
                 @Suppress("UNCHECKED_CAST")
-                startActivity(
+                val startParamss = StartActivityParams(
```

Comment:
this intermediate variable is redundant

Reply:
How to write if i need to convert the data type to StartActivityParams? I don't know how to write

Reply:
How about this?
`
if (shouldLaunchApp) {
                @Suppress("UNCHECKED_CAST")
                startActivity(
                    StartActivityParams(
                            parsedCaps["appPackage"] as? String,
                            parsedCaps["appActivity"] as? String,
                            parsedCaps["intentOptions"] as? Map<String, Any?>,
                            parsedCaps["activityOptions"] as? Map<String, Any?>)
            )
            }
`

Reply:
fine with me